packages/block-library/src/list/block.json

```diff
@@ -38,6 +38,8 @@
 	},
 	"supports": {
 		"anchor": true,
+		"align": [ "wide", "full" ],
+		"__experimentalLayout": true,
```

Comment:
I believe `__experimentalLayout` was stabilized to `layout` back in June 2023 via https://github.com/WordPress/gutenberg/pull/51434.